CLogger/include/CLogBuilder/CLogBuilder.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
+#include "stdafx.h"
+
+#include <memory>
+#include <list>
+
+#include "ELogConfig/ELogConfig.h"
+#include "CLogger/CLogger.h"
+#include "ELogLevel/ELogLevel.h"
+
+/// <summary>
+///		Namespace, that contains all classes and functions
+///		to work with CLogger and its macro-functions
+/// </summary>
+namespace Log
```

Comment:
I am not sure that we need this namespace

Reply:
Ok, it wiil be removed